locale/en/blog/community/2017-election.md

```diff
@@ -0,0 +1,83 @@
+The individual board members are able to provide feedback on how proposed board
+items might impact the community and can provide suggestions on how the Node.js
+Foundation can better support the community. With this knowledge and feedback,
+the board is able to contribute the resources the community needs to stay
+healthy and continue to grow.”  
+
+>> *Todd Moore, Node.js Foundation Director*
+
+The Individual Membership Director is responsible for soliciting feedback and
+data that represents the wishes of other individual members and the community at
+large. They have been entrusted with the duty to make decisions based on the
+information they receive to best represent the community, and can gather input
+for proposals when relevant and granted permission to do so.
+
+This includes participating and voting in Board meetings, introducing and
+driving forward initiatives to conclusion that capture the mission of the
+Node.js project, and representing the Board at speaking engagements (this is by
+no means a comprehensive list). Read more about [“What’s it like being on the
+Node.js Foundation Board of
+Directors?”](https://medium.com/@nodejs/whats-it-like-being-on-the-node-js-foundation-board-of-directors-f9456b8b7c4d).
+
+### What does the Board of Directors do?
+The Board meets every month to approve resolutions and discuss Node.js
+Foundation administrative matters. This includes legal considerations, budgeting
+and approving Foundation-led conferences and other initiatives. Technical
+governance is overseen by the TSC, not the Board of Directors.
+
+The current board members are listed
+[here](https://nodejs.org/en/foundation/board).
+
+### Who is running for the Individual Membership Director seat?
+Read more about why our candidates are running below.
+
+- William Kapke [@williamkapke](https://github.com/williamkapke)
+  - [I'm running for the Node.js Board of Directors!](https://www.youtube.com/watch?v=zPBOkqclJFc&feature=youtu.be)
+- Kat Marchán [@zkat](https://github.com/zkat)
+  - [Director nomination for Kat Marchán](https://gist.github.com/zkat/345d1485fc4cd1f45155678a3729cd21)
+- Charlie Robbins [@indexzero](https://github.com/indexzero)
+ - [I'm running to increase transparency to the community from the Node.js Board
+  of Directors](https://gist.github.com/indexzero/5b17885d8f8044b674a1fa411e1f7e3b)
+- William P. Riley-Land [@wprl](https://github.com/wprl)
+ - [I Would Like to Represent Individual Members of the Node.js Foundation](https://medium.com/@wprl/i-would-like-to-represent-individual-members-of-the-node-js-foundation-977157d90aa0#.hq3vo8d8m)
+
+### When is the election? Nominations were solicited until January 15th.
+- Ballot will be distributed on January 20th.
+- The election will close **January 30th at 17:00 UTC**.  
+
+### How do I vote?
+You must be an Individual Member of the Node.js Foundation to cast a vote. If
+you are a member, you can vote [NOW](vote.linuxfoundation.org)!
```

Comment:
Ah! No deeper link. The election will be front and center at that url, but does need `https://`